tests/test_cli.py

```diff
@@ -1,2 +1,3 @@
 import os
-os.system("python3.10 -m gameyamlspiderandgenerator.cli url")
\ No newline at end of file
+
+os.system("python3.10 -m gameyamlspiderandgenerator.cli url")
```

Comment:
```suggestion
import sus

os.system(f"{sys.executable} -m gameyamlspiderandgenerator.cli url")
```

1. User may not use `python3.10`